src/common/routes.ts

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import IRoute from "./interface";
+import * as pageRoutes from "./pageRoutes";
+//Lazy routes
+const HomePage = React.lazy(() => import("../Home/Home"));
+const AboutPage = React.lazy(() => import("../About/About"));
+const Auth = React.lazy(() => import("../Auth/Auth"));
```

Comment:
I wasn't able to find `src/Auth` in the codebase

Reply:
This hasn't been addressed

Reply:
this will be added later